.github/workflows/gluten.yml

```diff
@@ -55,20 +49,45 @@ jobs:
         id: gluten-cache
         uses: actions/cache@v4
         with:
-          path: gluten/package/target/
+          path: incubator-gluten/package/target/
           key: gluten_package_${{ steps.date.outputs.date }}
-      - name: Build gluten project
+      - name: Get Ccache
+        id: gluten-ccache-cache
+        uses: actions/cache@v4
+        with:
+          path: incubator-gluten/.ccache
+          key: ccache-centos7-release-default-${{github.sha}}
+          restore-keys: |
+            ccache-centos7-release-default
+      - name: Build Gluten velox third party
         run: |
           if [[ "${{ steps.gluten-cache.outputs.cache-hit }}" != 'true' ]]; then
-            git clone https://github.com/oap-project/gluten.git
-            cd gluten
-            ./dev/buildbundle-veloxbe.sh
+            git clone https://github.com/apache/incubator-gluten.git
+            cd incubator-gluten
+            docker run -v $GITHUB_WORKSPACE/incubator-gluten:/work -w /work apache/gluten:vcpkg-centos-7 bash -c "
```

Comment:
How hard would it be if we just built it from scratch?

Reply:
This docker image includes the gluten/velox dependencies, reducing the time to download and build dependencies.